extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanClientRuntimeConfig.java

```diff
@@ -186,14 +171,14 @@ public class InfinispanClientRuntimeConfig {
      * Whether a tracing propagation is enabled in case the Opentelemetry extension is present.
      * By default the propagation of the context is propagated from the client to the Infinispan Server.
```

Comment:
Technically this is no longer true as the "default" is what the binary has compiled in it with the client xml file.

Reply:
In Infinispan is true by default when the open telemetry extension is present when we don't configure the value...

Reply:
Yes, but I am talking about Quarkus Infinispan Client extension. So just the fact that it requires the presence of something also means true isn't the default value. But I was also referring to the fact that the runtime configuration is also dependent upon what the build time configuration is, since it can be defined in both places. I would just say remove the line to remove ambiguity.